docs/recipes/isolating-mongodb-integration-tests.md

```diff
@@ -0,0 +1,86 @@
+# Setting up Ava for Isolated MongoDB Integration Tests
+
+This recipe outlines how to run disposable MongoDB databases in your Ava tests, isolated between tests.
```

Comment:
You should say something like "this uses the `mongomem`" module and then provide a link to npm or GitHub.